lib/components/tasks/rotating_icon_button.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+class RotatingIconButton extends StatefulWidget {
+  RotatingIconButton({
+    super.key,
+    required this.turns,
+    required this.onPressed,
+  });
+
+  final void Function() onPressed;
+  double turns;
+
+  @override
+  State<RotatingIconButton> createState() => _RotatingIconButtonState();
+}
+
+class _RotatingIconButtonState extends State<RotatingIconButton> {
+  void _rotateIcon() {
+    setState(() {
+      widget.turns += 0.5;
```

Comment:
czy to w pewnym momencie sie nie wypierdoli? Może modulo robić z tego lepiej, chcoiaz wiadomo ze ktos musialby spędzić pół wieku wiec moze niepotrzebnie sie przypierdalam hahahah
![image](https://github.com/Bartosiki/DW-Flutter-App/assets/84678175/775dd5ff-1a90-48d2-a113-403240694eac)